src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,65 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Class implementing stack
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int NUMBERS_QUANTITY = 12;
+	private static final int EMPTY_STACK_INDICATOR = -1;
+	private static final int FULL_STACK_INDICATOR = 11;
```

Comment:
This constant is related to NUMBERS_QUANTITY. Changing only one leads to an error (inconsistency).